docs/reference/detr.md

```diff
@@ -328,16 +335,16 @@ Furthermore, demos on performing [training](../../projects/perception/object_det
   import numpy as np
   import urllib
   import cv2
-  from opendr.perception.object_detection_2d.detr.detr_learner import DetrLearner
-  from opendr.perception.object_detection_2d.detr.algorithm.util.draw import draw
-  
-  
+  from opendr.perception.object_detection_2d import DetrLearner
+  from opendr.perception.object_detection_2d import draw
```

Comment:
```suggestion
  from opendr.perception.object_detection_2d.detr.algorithm.util.draw import draw
```
The draw utility function is not specified in the __init__.py in object_detection, so that would result in an error.
It can be imported like this or is it better to add it to the __init__.py?

Reply:
You're right. I didn't add it to the `__init__` because there's 2 draw utilities, one for GEM and one for DETR, my bad.